src/util/renderers/article.renderer.ts

```diff
@@ -1,13 +1,40 @@
-import domino from 'domino'
 import { MWCapabilities } from '../../Downloader.js'
 import { Dump } from '../../Dump.js'
-import { DELETED_ARTICLE_ERROR } from '../const.js'
-import * as logger from '../../Logger.js'
-import { getStrippedTitleFromHtml } from '../misc.js'
+
+import { Renderer } from './abstractRenderer.js'
+import { ParsoidHtmlRestApiRenderer } from './parsoidHtmlRestApi.renderer.js'
+import { VisualEditorRenderer } from './visualEditor.renderer.js'
+
+export interface MWRendererArgs {
```

Comment:
I understand this is a first move, but this should actually be simpler (less arguments). We won't create a renderer for each article, so here it has to be changed.

Reply:
I passed the object with the necessary arguments to `render()` method itself. There should not be creation of the renderer each time, but only `render(args)` invocation.